bin/resources/shaders/dx11/convert.fx

```diff
@@ -357,12 +357,65 @@ PS_OUTPUT ps_convert_rgba_8i(PS_INPUT input)
 	return output;
 }
 
-// DUMMY
+cbuffer cb0
```

Comment:
Feels odd to put these here (cbuffer and rgb2yuv) , I'd have put them above, around line 40 would've made more sense.